docs/nx-cloud/reference/nx-cloud-cli.md

```diff
@@ -5,6 +5,12 @@
 At the beginning of your main job, invoke `npx nx-cloud start-ci-run`. This tells Nx Cloud that the following series of
 command correspond to the same CI run.
 
+{% callout type="warning" title="Do not run start-ci-run locally" %}
+`nx-cloud start-ci-run` is designed to run in CI. If you run the command locally, it can cause your local Nx repo think it's apart of a CI run. This can cause strange behavior like Nx commands timing out or throwing unexpected errors.
+
+If you ran the command locally, you can remove your systems temp files to reset your local system. Typically restarting your system is enough to fix this issue.
```

Comment:
@StalkAltan can you confirm this is the correct way to clear the system when start-ci-run is ran locally? couldn't remember if you also needed to nuke node_modules or not.

Reply:
This is fine, and it will be harder to run this locally very soon (requiring a --force flag)